export/export.h

```diff
@@ -31,8 +31,8 @@
 
 #ifndef MODULE_MONO_ENABLED
 
-#include "thirdparty/libnx/nacp.h"
-#include "thirdparty/libnx/nro.h"
+#include "platform/switch/thirdparty/libnx/nacp.h"
```

Comment:
This file should not contain any platform-specific code, so this kinda just feels... dirty. 😬